### PR TITLE
make notification container relative in mobile

### PR DESCRIPTION
**Pull request by shayanbinary**

### CODE REVIEW

File: app/_ciq-notification.scss [hunk 1]
```diff
@@ -82,3 +82,10 @@
     }
 }
 
+.smartcharts-mobile {
+    .cq-notifications {
+        position: relative;
+        right: 0px;
+        top: 0px;
```
Comment (MahboobehMohammadi) [inline app/_ciq-notification.scss]: @shayanbinary can we have something like a top:-25px so we remove the space between notification and the market selector? just a suggestion though.

Comment (shayanbinary) [reply app/_ciq-notification.scss]: @MahboobehMohammadi the problem is that `notification` div is under the `asset information` div if I move it up, it go on the asset information

Comment (MahboobehMohammadi) [reply app/_ciq-notification.scss]: @shayanbinary fair point.its okay though

**Comment by MahboobehMohammadi** [comment]
LGTM

File: app/_ciq-notification.scss (hunk 1 above)
Comment (shayanbinary) [reply app/_ciq-notification.scss]: @MahboobehMohammadi yeah you are right, the margin belongs to `comparison` container. which I remove it on mobile mode.

Comment (MahboobehMohammadi) [reply app/_ciq-notification.scss]: @shayanbinary yep. much better,thanks.

**Comment by MahboobehMohammadi** [comment]
LGTM.

**Pull request merged**